server/sandbox_run.go

```diff
@@ -63,6 +63,26 @@ func (s *Server) runContainer(container *oci.Container, cgroupParent string) err
 	return nil
 }
 
+func setNetworkReadiness(data interface{}, err error) {
+	s, ok := data.(*Server)
+	if !ok {
+		logrus.Errorf("invalid networking callback cookie")
```

Comment:
Suggest logging `data` here. Also you're using `Errorf` without any format parameters